### PR TITLE
Showing unapproved posts within unapproved topics

### PR DESCRIPTION
**Pull request by bodrovis**
Should fix https://github.com/radar/forem/issues/474 Tests do pass.

### CODE REVIEW

File: app/views/forem/moderation/index.html.erb [hunk 1]
```diff
@@ -9,14 +9,13 @@
       <%= render posts, :mass_moderation => true %>
     <% end %>
   </div>
-  <%= submit_tag t('forem.posts.moderation.moderate'), :class => "btn btn-primary" %>
```
Comment (radar) [inline app/views/forem/moderation/index.html.erb:12]: I don't understand why this button has been removed.

Comment (bodrovis) [inline app/views/forem/moderation/index.html.erb:12]: Post is being rendered with its own submit button, so for 2 posts I see 3 buttons which is strange

Comment (radar) [inline app/views/forem/moderation/index.html.erb:12]: @bodrovis can you show me a screenshot of what this used to look like?

Comment (bodrovis) [inline app/views/forem/moderation/index.html.erb:12]: Here you go. I have 1 unapproved post but still 2 buttons: one from this view and one from the partial with the post and its moderation tools:

**Comment by radar** [comment]
Added to rails3 and rails4. Thanks!

**Pull request closed without merging**